src/components/Articles.jsx

```diff
@@ -0,0 +1,29 @@
+import { useEffect, useState } from "react";
+import { getArticles } from "../api";
+
+const Articles = () => {
+  const [articles, setArticles] = useState([]);
+
+  useEffect(() => {
+    getArticles().then((articlesFromAPi) => {
+      console.log(articlesFromAPi, "<<<<<<<<");
+      setArticles(articlesFromAPi);
+    });
+  }, []);
+
+  return (
+    <main>
+      <h2>List of our artilces</h2>
+      {articles.map((article) => {
```

Comment:
You might want to render more of the information that is coming back from each artilce